<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QG_DlgScale</class>
 <widget class="QDialog" name="QG_DlgScale">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>277</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>190</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Scaling Options</string>
  </property>
  <widget class="Q3ButtonGroup" name="bgNumber">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>159</width>
     <height>165</height>
    </rect>
   </property>
   <property name="title">
    <string>Number of copies</string>
   </property>
   <layout class="QVBoxLayout">
    <item>
     <widget class="QRadioButton" name="rbMove">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>18</height>
       </size>
      </property>
      <property name="text">
       <string>&amp;Delete Original</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbCopy">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>18</height>
       </size>
      </property>
      <property name="text">
       <string>&amp;Keep Original</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbMultiCopy">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>18</height>
       </size>
      </property>
      <property name="text">
       <string>&amp;Multiple Copies</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leNumber"/>
    </item>
    <item>
     <spacer name="spacer7">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>16</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>237</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="_2">
    <item>
     <widget class="QLabel" name="lFactor_5">
      <property name="text">
       <string>x</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="scaleFactorX">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>Scale factor at the x direction</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lFactor_4">
      <property name="text">
       <string>y</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="scaleFactorY">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>Scale factor for the y direction</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>224</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <item>
     <spacer name="spacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="bOk">
      <property name="text">
       <string>&amp;OK</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bCancel">
      <property name="text">
       <string>&amp;Cancel</string>
      </property>
      <property name="shortcut">
       <string>Esc</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>10</y>
     <width>86</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../../../res/extui/extui.qrc">
     <normaloff>:/extui/dlgmodifyscale.png</normaloff>:/extui/dlgmodifyscale.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>237</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <item>
     <widget class="QLabel" name="lFactor">
      <property name="text">
       <string>&amp;Factor (f):</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="isotropic">
      <property name="toolTip">
       <string>Scale by the same factor at both x- and y- direction</string>
      </property>
      <property name="text">
       <string>Isotropic Scaling</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="cbCurrentLayer">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>210</y>
     <width>146</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Use current &amp;layer</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbCurrentAttributes">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>190</y>
     <width>179</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Use current &amp;attributes</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends>Q3GroupBox</extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">rs_modification.h</include>
 </includes>
 <resources>
  <include location="../../../res/extui/extui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>rbMove</sender>
   <signal>toggled(bool)</signal>
   <receiver>leNumber</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbCopy</sender>
   <signal>toggled(bool)</signal>
   <receiver>leNumber</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbMultiCopy</sender>
   <signal>toggled(bool)</signal>
   <receiver>leNumber</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bOk</sender>
   <signal>clicked()</signal>
   <receiver>QG_DlgScale</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bCancel</sender>
   <signal>clicked()</signal>
   <receiver>QG_DlgScale</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
